docker-entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+
+# first check if we're passing flags, if so
+# prepend with memcached
+if [ "${1:0:1}" = '-' ]; then
+	set -- memcached "$@"
```

Comment:
This entrypoint script allows starting the image with args like this:

`docker run memached -l 0.0.0.0`

Without it, we can still pass args, like this:

`docker run memached memcached -l 0.0.0.0`